src/lists.c

```diff
@@ -2807,6 +2807,38 @@ static Int CheckInit (
     /* check implications                                                  */
     for ( i = FIRST_LIST_TNUM;  i <= LAST_LIST_TNUM;  i++ ) {
 
+        if ( (i & IMMUTABLE) == 0 ) {
+            if ( ClearFiltsTNums[i]+IMMUTABLE != ClearFiltsTNums[i+IMMUTABLE]) {
+                Pr( "#W  ClearFiltsTNums [%s] mismatch between mutable and immutable\n",
+                    (Int)(InfoBags[i].name), 0 );
+                success = 0;
+            }
+            for ( j = 0;  j < ARRAY_SIZE(fnums);  j++ ) {
```

Comment:
Should the remaining loops in `CheckInit` then also be functions?

Anyaway: strictly speaking, I bope these three loops would soon be removed again anyway, as part of PR #1563, as there'd be no more IMMUTABLE bit in the TNUM...